Source/RMG/UserInterface/Widget/RomBrowserWidget.cpp

```diff
@@ -95,13 +95,13 @@ RomBrowserWidget::RomBrowserWidget(QWidget *parent) : QStackedWidget(parent)
     this->listViewWidget->horizontalHeader()->setDefaultAlignment(Qt::AlignLeft);
     this->listViewWidget->horizontalHeader()->setSortIndicatorShown(false);
     this->listViewWidget->horizontalHeader()->setHighlightSections(false);
-    this->listViewWidget->horizontalHeader()->setContextMenuPolicy(Qt::ContextMenuPolicy::CustomContextMenu);
+    this->listViewWidget->horizontalHeader()->setContextMenuPolicy(Qt::CustomContextMenu);
```

Comment:
Please revert `Qt::CustomContextMenu` back to `Qt::ContextMenuPolicy::CustomContextMenu);`

Reply:
Done, will include in next commit.